readme.md

```diff
@@ -87,9 +87,6 @@ pageres --delay 3 1366x768 < urls.txt
 
 # Capture a local file
 pageres unicorn.html 1366x768
-
-# Pipe in resolutions
-cat screen-resolutions.txt | pageres todomvc.com yeoman.io
```

Comment:
Yes, but I always found it awkward tbh. Better to just use string interpolation in the command.